### ISSUE TITLE
"I Downvoted YOU Because" encourages people to take things personally

### ISSUE THREAD

**Issue by Aralun** [WillSullivan/IDownvotedYouBecause#6]
I'm mainly active on Stack Overflow, and one of the main issue I see when voting or moderating is how personal people get. I always downvote/vote to close the post itself, judging it solely on its content and not on whoever posted it.

Thing is, receiving a downvote or seeing your post closed is very unpleasant. People get angry, wonder why some high-rep scumbags are bullying them, feel helpless, ...
This can become more of an issue when a post is closed by some people in the [SOCVR chat room](https://chat.stackoverflow.com/rooms/41570/so-close-vote-reviewers) and users feel like they, as a person, have been the target of organized attacks.

I generally have a couple auto-comments I leave to explain why I acted and what can be done (if possible) to improve the situation of the post. Thing is, comments only go so far, and a resource like IDYB would be _freaking awesome_ to provide more data to users.

But, the title.

I am most concerned about is this word: **"YOU"**.
I downvoted **YOU**. **YOU** did something bad, **YOU** do not deserve my knowledge, and so on.

What is a user who just saw his question closed and downvoted to -6 going to think when reading that? 

I do not downvote users, but posts. It'd be pretty damn great if this could figure in the title.
Something like **"I Downvoted This Because..."**, or anything that makes it less personal.

**Comment by WillSullivan** [comment]
I can understand and appreciate what you're saying here.  In fact, one of the goals of this website is to change what could easily (and often does) be taken personally as an attack (_I_ am being downvoted) and try to turn that into a positive learning experience.  

But I definitely **don't** want to sanitize things for the downvoter, as that could be taken as an attempt on the part of the downvoter to make _them_ look like a good samaritan, and kill the conversation before it even happens.
- "You downvoted me!  Why did you do that, you jerk!  This is important!"
- "No, I didn't downvote _you_, I downvoted this question of yours.  See, I don't hate you, I hate your post.  Here's a link you can use to fix it.  I'll remove my downvote if you do."
- "You... Wow, thanks, hero."

It just smells bad. It negates the downvotee's perception while making the downvoter look like they didn't do anything that could be perceived as bad.  While from the downvoter's perspective that may feel good, if we're trying to **help** the downvoted, this shouldn't be even considered.
- "You downvoted me!  Why did you do that, you jerk! This is important"
- "Yep. Here's why, and what you can do to fix it.  I'll remove my downvote if you do."
- "You... Wow, thanks, hero."

To me, the last line sounds completely different now.  What do you think?

Anyhow, I'll leave this open for discussion.

**Comment by gunr2171** [comment]
This issue revolves around two aspects: the perception of the downvotee, and the behavior of the downvoter.

Removing the "You" from the title will help to avoid the downvotee assuming it's a personal attack. However, this only works if the downvoter keeps the same intent.

I'm a Room Owner of the [SOCVR room](http://chat.stackoverflow.com/rooms/info/41570/so-close-vote-reviewers) (Aralun is one of the regulars). When our room started we were just 3 members hoping to make a dent in the CV queue. Now, we've got 10+ regulars and have gained some serious power. We can close any question on-demand.

This is a serious responsibility and had a [big meeting](http://chat.stackoverflow.com/rooms/41570/conversation/november-2015-room-meeting) recently to address it. The big take away is to ensure that we don't ever target a user. Aralun talked at length about that here in the first post.

Along with addressing how we close vote, the other main aspect from the meeting was user communication behavior, whether in the comments or in chat.

> How we say something, especially in an environment where it is difficult to correctly convey emotions, is critical.
> We should be thrilled that a user hopped in chat. Now you have a chance to guide them, not criticize them to the ground. Don't show them the error of their ways, show them how to get back on the path.
> This won't be an easy thing; we know first hand how much unsalvageable content is on the site. Just remember that everyone who posts stuff here is human.
> Not everyone is Jon Skeet, but you should treat them like they were.

In your first dialog example:

> - "You downvoted me! Why did you do that, you jerk! This is important!"
> - "**No, I didn't downvote you, I downvoted this question of yours. See, I don't hate you, I hate your post.** Here's a link you can use to fix it. I'll remove my downvote if you do."
> - "You... Wow, thanks, hero."

Yes, it's easy to take a pure defencive stance when someone accuses you of doing something wrong. However, it should be our goal to avoid argument, and focus first on editing the content.

Instead of a defensive stance for the downvoter, they could have a positive attitude.

> "I downvoted your post because it's X, Y, and Z. Here's a link you can use to fix it. I'll remove my downvote if you do."  
> "Here's a link you should read through. It will help you fix some problems in your post, like X and Y."

The rules of Stack Overflow is to downvote bad content. Don't punish the downvoter for that, but they need to be accountable for user interactions if they comment.

To sum it up: I think the "you" should be dropped, but the downvoter needs to be the bigger person. We can't expect this site to fix that behavior, but we can get halfway there.

**Comment by WillSullivan** [comment]
(sorry, been super busy lately and super lazy during the holiday break. Intend to give this the attention it deserves as time becomes available)

**Comment by onebree** [comment]
Why not just rename the site `downvoted` or `downvote-reasons`? Simple enough meaning, without personally attacking the voter/asker.

**Comment by WillSullivan** [comment]
Because I don't think it is a personal attack against the downvoter.  

But people can take getting downvoted as a personal attack.  And if you're the one they're angry at, trying to use weasel terms to insulate you from your actions never works.  In fact, it often has a negative effect.  That's the perspective from which I wanted to work.  I want the response to be "Yes, and.." not "No, but...".  

I should create another website called "IDontLikeWhatYouCalledYourWebsiteBecause.com", or maybe "YourWebsiteIsNamedPoorlyAndThisIsWhy.com" or "ItsNotMeItsYourWebsitesName.com".

**Comment by WillSullivan** [comment]
This project has languished, so I've decided to give it a go.  Bought idownvotedbecause.com and will migrate over in the coming week (as time permits).

What the heck, I'll see how things turn out.  Hopefully this will attract a little more participation.  I doubt it will hurt anything.  Except my ego.  A little bit.  Not much, just a smidge.

I'm going to take another stab at using jeckyll to compile the website.  I've figured out I can avoid Posts (with their date-based urls), so I'll try that.  If it works well, I might mothball this repository.

**Comment by Aralun** [comment]
How's the migration going? I'd like to integrate it into [SOCVR auto-comments](https://github.com/SO-Close-Vote-Reviewers/auto-comments).

**Comment by WillSullivan** [comment]
It's on the schedule to start this week.  I've got some IRL heavy lifting to finish before I can spend some time on this.  I'm hoping to start work on the Jenkins site this weekend, but I won't really have much time to spend on it for the next two, two and a half weeks.

**Comment by WillSullivan** [comment]
Okay, worked on getting the initial site working with Jekyll over the weekend (I was struggling with posts when I should have been making pages, ugh) over at idownvotedbecause.com.  Forgot to sync until just now.  It needs some skinning, but that's where work will be taking place.  I've already migrated over the image of an exception page, and will migrate the rest later.
